notebooks/Community.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NetworKit Community Detection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we will cover some algorithms on the `community` module. Community detection is concerned with identifying groups of nodes which are significantly more densely connected to eachother than to the rest of the network. The first step is to import NetworKit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import networkit as nk"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `community` module provides a top-level function, [detectCommunities(G, algo=None, inspect=True)](https://networkit.github.io/dev-docs/python_api/community.html?highlight=detect#networkit.community.detectCommunities) to quickly perform community detection of a given with a suitable algorithm and print some statistics about the result. If we pass a graph to the this function, it choses a suitable algorithm and can be used as follows:"
```

Comment:
The first sentence is a bit confusing. Which algorithm does this function run? How is it chosen? Does quickly mean that the algorithm is fast?
The second sentence is redundant, you can simply write "This function can be used as follows:"